DOCS/man/options.rst

```diff
@@ -1664,21 +1664,16 @@ Video
 
         Works in ``--no-correct-pts`` mode only.
 
-``--deinterlace=<yes|no>``
-    Enable or disable interlacing (default: no).
+``--deinterlace=<yes|no|auto>``
+    Enable or disable interlacing (default: auto).
     Interlaced video shows ugly comb-like artifacts, which are visible on
     fast movement. Enabling this typically inserts the bwdif video filter in
     order to deinterlace the video, or lets the video output apply deinterlacing
     if supported.
 
-    This behaves exactly like the ``deinterlace`` input property (usually
-    mapped to ``d``).
-
-    Keep in mind that this **will** conflict with manually inserted
-    deinterlacing filters, unless you take care. (Since mpv 0.27.0, even the
-    hardware deinterlace filters will conflict. Also since that version,
-    ``--deinterlace=auto`` was removed, which used to mean that the default
-    interlacing option of possibly inserted video filters was used.)
+    The default value of ``auto`` will deinterlace if mpv detects taht the video
+    is interlaced and disable the filter if it is not interlaced. Keep in mind
```

Comment:
Would be nice to mention how this detection works. Container metadata? Codec data? I recognize you only know insofar mpv knows somehow, but documenting how mpv knows would help users debug issues where it is either not auto-detected or falsely detected to be interlaced.

Reply:
Well the detection is just "ffmpeg says so". Slightly reworded it to hopefully be clearer here.